testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/Suite2.java

```diff
@@ -36,6 +36,7 @@ public List<SuiteTestRun> getTestRuns(EnvironmentConfig config)
         return ImmutableList.of(
                 testOnEnvironment(EnvMultinode.class)
                         .withGroups("configured_features", "hdfs_no_impersonation")
+                        .withExcludedTests("io.trino.tests.product.TestImpersonation.testExternalLocationTableCreationSuccess")
```

Comment:
why this exclusion?

Reply:
to be honest, don't remember any specific reason, need to run and check this test

Reply:
understood. please recover this info & capture it as a code comment.

Reply:
> understood. please recover this info & capture it as a code comment.

got it I didn't want enable` hive.non-managed-table-writes-enabled` for whole `EnvMultinode` environment

Reply:
https://github.com/trinodb/trino/pull/20936